### PR TITLE
nixpkgs/brlaser: upgrade to v6

### PR DESCRIPTION
**Pull request by jakubgs**
### Why

It introduces support for several additional Brother printers:
```
Brother HL-2030 series
Brother HL-2140 series
Brother HL-2220 series
Brother HL-2270DW series
Brother HL-5030 series
Brother DCP-L2520D series
```
https://github.com/pdewacht/brlaser/releases/tag/v6

### Testing

I've tested this on Linux with HL-2140 and it works without any issues.

### Notify maintainers

@StijnDW

### CODE REVIEW

**Review by Stekke — APPROVED** [review]
Thanks for the pr :)

**Comment by jakubgs** [comment]
What's missing to merge this?

**Comment by jakubgs** [comment]
Doe we need another reviewer?

**Comment by StijnDW** [comment]
Nothing is missing as far as I know.
Someone from the Nixpkgs committers team will merge this when they have time.

**Comment by bhipple** [comment]
Result of `nixpkgs-review pr 79246` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
<br>- brlaser
</details>

**Review by bhipple — APPROVED** [review]

**Comment by bhipple** [comment]
LGTM, despite the "major version" upgrade the changelog appears to imply just adding new features, and it has no reverse dependencies, so :+1:

**Pull request merged**
